src/scss/03-widgets/_button-group.scss

```diff
@@ -149,3 +149,20 @@
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.button-group-item {
+		&.button-group-selected-item:before {
+			content: '';
+			display: block;
+			position: absolute;
+			top: 0;
+			right: 0;
+			bottom: 0;
+			left: 0;
+			border: 3px solid #b7b7b7;
```

Comment:
```suggestion
			border: var(--border-size-l) solid #b7b7b7;
```

Reply:
Please review the color used to use one of the available css variables

Reply:
Done!